contracts/serverless-v2/Executors.sol

```diff
@@ -269,7 +273,6 @@ contract Executors is
 
         if(executors[operator].activeJobs == 0) {
```

Comment:
we can add condition where unstaking is not allowed if de-registration is going on.